utils/middlewares/__tests__/magic-link-jwe-validation-middleware.test.ts

```diff
@@ -89,20 +79,15 @@ describe("MagicLinkJweValidationMiddleware", () => {
       .setIssuer(jweIssuer)
       .setIssuedAt()
       .setExpirationTime(getUnixTime(subSeconds(today, 3000)))
```

Comment:
Test near the validation interval
```suggestion
      .setExpirationTime(getUnixTime(subSeconds(today, 1)))
```